### PR TITLE
Feature/dm 3113 v0.2.0

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* add ganglia metric collection (https://<jenkins-master>/ganglia/)
+* fix postbuild script intended to terminate zombie lsstsw/rebuild processes
+* add hiera-eyaml support for mixed plaintext/encypted values in `common.eyaml`
+* add TLS support and a public CA signed keypair for *.lsst.codes
+* add jenkins-ec2-snapshot job to make EBS snapshots (backups) of jenkins-master
+* add jenkins purge-build-queue-plugin plugin; removes all pending builds
+* the AWS_REGION env var has been renamed to AWS_DEFAULT_REGION
+* http/s redirection to canonical site name
+* assorited usability and doc improvements
```
Comment (timj) [inline CHANGELOG.md]: Typo "assorited"

File: README.md [hunk 2]
```diff
@@ -89,6 +89,35 @@ Profiles](http://www.craigdunn.org/2012/05/239/) pattern.
   * The puppet modules in use are listed in the [Puppetfile](./Puppetfile).
   * The site module is under the [`jenkins_demo`](./jenkins_demo) directory.
 
+### metric collection
+
+A full production deployment would likely involved additional jenkins plugins
```
Comment (timj) [inline README.md]: Typo: involved -> involve

**Comment by jhoblitt** [comment]
Eh -- I swear I spell checked it... fixed.

File: README.md [hunk 3]
```diff
@@ -157,47 +184,90 @@ Sandbox
...
 ```
 
-    export CENTOS6_AMI=ami-92ccd2fa
-    export CENTOS7_AMI=ami-feccd296
+### Declare new AMI IDs
+
+To be used by vagrant.
+
+    export CENTOS6_AMI=ami-3331f958
+    export CENTOS7_AMI=ami-3331f958
```
Comment (timj) [inline README.md]: It doesn't seem that these should have the same value.

Comment (jhoblitt) [inline README.md]: The packer output above is duplicated as well.  Fixed.

**Pull request merged**